deploy/examples/create-external-cluster-resources.py

```diff
@@ -1299,6 +1299,11 @@ def validate_rbd_pool(self):
                 f"The provided pool, '{self._arg_parser.rbd_data_pool_name}', does not exist"
             )
 
+    def enable_rbd_pool(self):
```

Comment:
To clarify, please update the commit and PR title with "init" instead of "enable". 
```suggestion
    def init_rbd_pool(self):
```

Reply:
Updated